tests/cephadm/test_ceph_orch_for_monitoring_image_version.py

```diff
@@ -0,0 +1,52 @@
+import re
+from json import loads
+
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError
+
+
+def run(ceph_cluster, **kw):
+    """
+    Validate if the ceph orch ps command reflects the version
+    for monitoring stack images
+    """
+    config = kw["config"]
+    installer_node = ceph_cluster.get_nodes(role="installer")[0]
+    cephadm = CephAdm(installer_node)
+    monitoring_images = config.get("monitoring_images")
+
+    # Get the version using podman command
+    prometheus_img = monitoring_images["ose-prometheus"]
+    alertmanager_img = monitoring_images["ose-prometheus-alertmanager"]
+    node_exporter_img = monitoring_images["ose-prometheus-node-exporter"]
+
+    # Get the version using podman command
+    cmd = f'podman run -it --rm --user root --entrypoint /bin/bash {prometheus_img} -c "prometheus --version"'
```

Comment:
Instead of podman command use containers library - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/containers.py

Reply:
Done